README.md

```diff
@@ -1,38 +1,201 @@
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![LinkedIn][linkedin-shield]][linkedin-url]
+
+
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+ 
+
+  <h3 align="center">KT</h3>
+
+  <p align="center">
+   Track 2- News homepage solution
+    <br />
+    <a href="https://github.com/lo0oleta/news-homepage"><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://github.com/lo0oleta/news-homepage">View Demo</a>
+    ·
+    <a href="https://github.com/lo0oleta/news-homepage/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/lo0oleta/news-homepage/issues">Request Feature</a>
+  </p>
+</div>
+
+
+
+<!-- TABLE OF CONTENTS -->
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+      <ul>
+        <li><a href="#built-with">Built With</a></li>
+      </ul>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#installation">Installation</a></li>
+      </ul>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#roadmap">Roadmap</a></li>
+    <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#license">License</a></li>
+    <li><a href="#contact">Contact</a></li>
+    <li><a href="#acknowledgments">Acknowledgments</a></li>
+  </ol>
+</details>
+
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+<img width="1696" alt="image" src="https://user-images.githubusercontent.com/8960757/230594630-42c220e3-ec5e-4121-9bde-044d8f23b701.png">
+
+
+This is a frontendmentor challenge that tests your ability to create dropdown navigation menus, a common pattern on larger sites,
+It will also provide some nice basic layout challenges.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+### Built With
+
+This section should list frameworks/libraries used in the project.
+
+* [![Next][Next.js]][Next-url]
+* [![React][React.js]][React-url]
+* [![Chakra-ui][chakra.ui]][Chakra-url]
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
 ## Getting Started
 
-First, run the development server:
+This is an example of how you may give instructions on setting up your project locally.
+To get a local copy up and running follow these simple example steps.
+
+### Installation
+
+_Below is an example of how you can instruct your audience on installing and setting up your app. This template doesn't rely on any external dependencies or services._
+
+1. Clone the repo
+      ```
+
+   git clone https://github.com/lo0oleta/news-homepage.git
+   ```
+3. Install Yarn packages
+   ```
+
+   yarn
+    ```
+
+4. Run the development server
+   ```npm run dev
+     # or
+     yarn dev
+     # or
+     pnpm dev
+     
+   ```
+5.Open http://localhost:3000 with your browser to see the result.
```

Comment:
add a space after `5.` so that it can be identified as a part of the numbered list